src/APL/fileAPL.ts

```diff
@@ -0,0 +1,87 @@
+import debugPkg from "debug";
+import { promises as fsPromises } from "fs";
+
+import { APL, AuthData } from "./apl";
+
+const debug = debugPkg.debug("FileAPL");
+
+export const loadDataFromFile = async (fileName: string): Promise<AuthData | undefined> => {
+  debug(`Load auth data from the ${fileName} file`);
+  let parsedData: Record<string, string> = {};
+  try {
+    await fsPromises.access(fileName);
+    parsedData = JSON.parse(await fsPromises.readFile(fileName, "utf-8"));
+  } catch (err) {
+    debug(`Could not read auth data from the ${fileName} file`, err);
+    return undefined;
+  }
+  const { token, domain } = parsedData;
+  if (token && domain) {
+    return { token, domain };
+  }
+  return undefined;
+};
+
+export const saveDataToFile = async (fileName: string, authData?: AuthData) => {
+  debug(`Save auth data to the ${fileName} file`);
+  const newData = authData ? JSON.stringify(authData) : "{}";
+  try {
+    await fsPromises.writeFile(fileName, newData);
+  } catch (err) {
+    debug(`Could not save auth data to the ${fileName} file`, err);
+  }
+};
+
+export type FileAPLConfig = {
+  fileName?: string;
+};
+
+/**
+ * File APL
+ *
+ * The APL store auth data in the json file.
+ *
+ * Before using this APL, please take in consideration:
+ *   - only stores single auth data entry (setting up a new one will overwrite previous values)
+ *   - it's not recommended for production use - redeployment of the application will override
+ *     existing values, or data persistence will not be guaranteed at all depending on chosen
+ *     hosting solution
+ *
+ */
+export class FileAPL implements APL {
+  private fileName: string;
+
+  constructor(config: FileAPLConfig = {}) {
+    this.fileName = config?.fileName || ".auth-data.json";
```

Comment:
How about `.saleor-app-auth.json`? 

1. Having maaany files you are sure this is Saleor, there can be many other "auths"
2. `Data` is forbidden keyword meaning nothing in programming ;)